src/attackmate/__main__.py

```diff
@@ -137,38 +136,52 @@ def parse_playbook(playbook_file: str, logger: logging.Logger) -> Playbook:
     Playbook
         The parsed AttackMate playbook object
     """
+
+    # Get the path of the current working directory
+    current_working_directory = Path.cwd()
+
+    # Construct the path to the default playbooks directory
+    default_playbook_location = Path('/etc/attackmate/playbooks')
+
+    playbook_file_path = Path(playbook_file)
+
+    # 1 # Check provided path
+    if playbook_file_path.exists():
+        target_file = playbook_file_path
+
+    # 2 # Check current working directory
+    elif (current_working_directory / playbook_file_path).exists():
+        target_file = current_working_directory / playbook_file_path
+
+    # 3 # Check default playbook directory
+    elif (default_playbook_location / playbook_file_path).exists():
+        target_file = default_playbook_location / playbook_file_path
+
+    else:
+        logger.error(
+            f"Error: Playbook file not found under '{playbook_file_path}' or in the current directory or in /etc/attackmate/playbooks"
+        )
+        exit(1)
+
```

Comment:
Maybe we could add here `logger.info(f"Loading playbook file {target_file}")`
(could be helpful to know which one is used in the end, if there are files with the same name at more than one of those locations)

Reply:
Good idea @annaerdi 
Maybe with logger.debug, so that we are able to debug it if we need to debug the location

Reply:
thanks! added the debug logging for target_file